lighthouse-core/lib/network-request.js

```diff
@@ -9,6 +9,31 @@
  * @fileoverview Fills most of the role of NetworkManager and NetworkRequest classes from DevTools.
  * @see https://cs.chromium.org/chromium/src/third_party/blink/renderer/devtools/front_end/sdk/NetworkRequest.js
  * @see https://cs.chromium.org/chromium/src/third_party/blink/renderer/devtools/front_end/sdk/NetworkManager.js
+
+  DevTools box-whisker
```

Comment:
Between this documentation effort and my other goal of adding tests to chromium to assert resourcetiming & cdp & trace events all agree with eachother.... I really needed to determine and document the network timing dealio.

I went real overkill, but ... at least i almost covered everything I was hoping to..


![Network-Timings](https://user-images.githubusercontent.com/39191/182498804-b2260705-c990-4ba1-8167-fa42fe6f7cd5.svg)